nodejs/week3/warmup/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+node_modules/
```

Comment:
because of trust issues(past mistakes to upload node modules in github)  I wrote this command ' echo node_modules >> .gitignore' two times 😁 so it is